docs/content/en/docs/implementing/day-2-operations/_index.md

```diff
@@ -0,0 +1,295 @@
+---
+title: Day 2 Operations with Keptn
+description: How to operate and maintain your Keptn Apps
+weight: 500
+---
+
+After you have successfully rolled out your application by following
+the instructions in the [integration guide](../integrate),
+Keptn also assists you with day 2 operations for your application.
+
+Tasks that fall under this category include:
```

Comment:
you could link the following points to the different subsections that explain them so that users can easily jump right to where they wanna go